ember-aria-utilities/package.json

```diff
@@ -85,7 +85,7 @@
     "rollup-plugin-ts": "2.0.7",
     "semantic-release": "^19.0.2",
     "prettier": "^2.0.0",
```

Comment:
Are we electing not to pin dependencies on our open source stuff?

Reply:
correct -- this allows more aggressive "auto-maintenance" while also ensuring wider range of compatibility with whatever we use internally.

given a wide range of `^4.0.0`, internally, we can be on "whatever version", and to bump that (if it were a dependency version), would be a breaking change, as we'd be declaring dropped support for `4.0.0`.

additionally, we have extra re-assurance of non-breakage, due to more robust C.I.:  https://github.com/CrowdStrike/ember-aria/actions/runs/2416524904

specifically, we have many "scenarios" which are testing supported ember versions and typescript versions, and then a test that deletes the whole lockfile and re-generates it (the lockfile is used strictly in all C.I. jobs). 
Allowing a floating-deps test allows us to catch potential configuration issues and bugs in the broader JS ecosystem before they'd trickle into our internal code / updates.